std/algorithm/searching.d

```diff
@@ -3462,6 +3462,59 @@ unittest
 }
 
 /**
+Computes the first occurrence of `range`'s minimum (respectively maximum).
+
+Params:
+    pred = The ordering predicate to use to determine the extremum (minimum or
+           maximum) element.
+    range = The input range to search.
+
+Returns:
+    The position of the first encounter of the minimum (respectively maximum) in `range`.
```

Comment:
I think _index_ is better than _position_, and we should mention -1 when the range is empty.